metsim/constants.py

```diff
@@ -0,0 +1,56 @@
+"""
+Stores default constants, parameters, and options.  
+"""
+
+SECPERRAD= 13750.9871  # seconds per radian of hour angle
+RADPERDAY= 0.017214    # radians of Earth orbit per julian day
+RADPERDEG= 0.01745329  # radians per degree
+MINDECL= -0.4092797    # minimum declination (radians)
+DAYSOFF= 11.25         # julian day offset of winter solstice
+SRADDT= 30.0           # timestep for radiation routine (seconds)
+                       #    Note:  Make sure that 3600 % SRADDT == 0
+MA= 28.9644e-3         # (kg mol-1) molecular weight of air
+R= 8.3143              # (m3 Pa mol-1 K-1) gas law constant
+G_STD= 9.80665         # (m s-2) standard gravitational accel.
+P_STD= 101325.0        # (Pa) standard pressure at 0. m elevation
+T_STD= 288.15          # (K) standard temp at 0. m elevation
+CP= 1010.0             # (J kg-1 K-1) specific heat of air
+LR_STD= 0.0065         # (-K m-1) standard temperature lapse rate
+ 
+KELVIN= 273.15
+EPS= 0.62196351
+DAYS_PER_YEAR = 365.25
+SEC_PER_DAY   = 86400
+MIN_PER_HOUR  = 60
+HOURS_PER_DAY  = 24
+SEC_PER_HOUR  = 3600
+STEFAN_B = 5.669e-8     # (W m^-22 K^-4) Stefan Boltzmann constant 
+TDAYCOEF= 0.45          # (dim) daylight air temperature coefficient
+
+BAR_TO_MBAR = 1./1000
+CM_TO_MM = 1./10
```

Comment:
Confusing ... since 1 bar equals 1000 mbar and 1 cm equals 10 mm, these seem to be exactly opposite to what their name indicates (assuming this is a multiplication, not a division). At the very least provide a use case in a comment.